src/help/constants.ts

```diff
@@ -88,5 +88,30 @@ export const helpAndFeedbackItems: HelpAndFeedbackItem[] = [
     id: 'aiAgentsConfigurationDoc',
     url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/ai-capabilities/agents#sonarqube-mcp-server',
     viewItem: false
+  },
+  {
+    id: 'earlyAccessTerms',
+    url: 'https://www.sonarsource.com/legal/early-access/',
+    viewItem: false
+  },
+  {
+    id: 'privacyNotice',
+    url: 'https://www.sonarsource.com/company/privacy/',
+    viewItem: false
+  },
+  {
+    id: 'preCommitAnalysisLink',
+    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/getting-started/running-an-analysis/#analyze-changed-files',
+    viewItem: false
+  },
+  {
+    id: 'mcpIntegrationLink',
+    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/ai-capabilities/agents#sonarqube-mcp-server',
+    viewItem: false
+  },
+  {
+    id: 'dependencyRiskManagementLink',
+    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/using/dependency-risks',
```

Comment:
Not sure about its importance, on all other places where the docs links are used, they is a `/` appended. Also should the docs links be replaced with values from the constants? 🤔 

```suggestion
    id: 'preCommitAnalysisLink',
    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/getting-started/running-an-analysis/#analyze-changed-files/',
    viewItem: false
  },
  {
    id: 'mcpIntegrationLink',
    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/ai-capabilities/agents#sonarqube-mcp-server/',
    viewItem: false
  },
  {
    id: 'dependencyRiskManagementLink',
    url: 'https://docs.sonarsource.com/sonarqube-for-vs-code/using/dependency-risks/',
```